docs/containers.conf.5.md

```diff
@@ -205,6 +205,8 @@ Run an init inside the container that forwards signals and reaps processes.
 
 **init_path**="/usr/libexec/podman/catatonit"
 
+Deprecated: It is recommend to place catatonit in a directory listed under the **helper_binaries_dir** option.
```

Comment:
What does deprecation mean in this context?  Is it a NOP from now on or will it still be used if set? (Didn't read the Podman PR yet but I think users may have the same question)

Reply:
It still works fine, same for the other undocumented field under the engine section, see FindInintBinary().
With deprecated I mean that users should instead of setting this field place it under one of the helper binaries directories instead. This field will continue to work and I have no intention of removing it as this will break backwards compat.

Reply:
Could we rephrase "deprecated" to "We recommend using $the_other_thing instead"?  I feel that "deprecated" raises more questions than answers in this specific case.

Reply:
I prefer to remove deprecated stuff from the man page totally,  If the user has a containers.conf with this field, it will be used but we don't need to menetion it in man page or containers.conf.

Reply:
Sure, I went with deprecated because that leaves the door open to remove it at some point. But I guess both fields are simple enough to keep them supported even for future major versions.